pallets/capacity/src/lib.rs

```diff
@@ -904,79 +912,100 @@ impl<T: Config> Pallet<T> {
 		CapacityLedger::<T>::insert(target, capacity_details);
 	}
 
-	fn get_staking_type_and_releasable_percent_in_force(
-		current_staking_type: StakingType,
+	/// Get the phase of Committed Boosting at the indicated block
+	fn get_committed_boosting_phase(
```

Comment:
I was surprised there wasn't a nice unit test for this. Suggestion that it might be worth having an AI generate a set to cover the cases as a separate set of tests instead of just testing a level up.

Reply:
Tests are coming 😄

Reply:
Once I've got the final functional updates merged into the feature branch, I'm going to do a unit & e2e test PR.